app/database/accessor/breeze_buddy/outbound_number.py

```
@@ -297,6 +300,36 @@ async def get_outbound_number_based_on_status_and_provider(
+        )
+        result = await run_parameterized_query(query_text, values)
+
+        if result and get_row_count(result) > 0:
+            decoded_result = decode_outbound_number(result)
+            logger.info(f"Outbound number IVR config updated: {decoded_result}")
+            return decoded_result
+
+        logger.error(
+            f"Failed to update IVR config for outbound number ID: {outbound_number_id}"
+        )
+        return None
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)